proxy/core/src/main/resources/META-INF/native-image/io.cloudstate/cloudstate-proxy-core/cloudstate-reflect-config.json.conf

```diff
@@ -0,0 +1,59 @@
+[
+  {
+    name: "io.cloudstate.proxy.autoscaler.KubernetesDeploymentScaler$Metadata"
```

Comment:
I'm not sure I understand the WHY of this PR, but if we are switching to HOCON, would it not make sense to switch to a less verbose format?

Reply:
(answering in the main thread)